static/styles/subscriptions.css

```diff
@@ -320,16 +320,16 @@ h4.stream_setting_subsection_title {
             color: inherit;
         }
     }
+
+    .form-inline {
+        margin-bottom: 0;
+    }
 }
 
 .subscriber-search {
     margin: 10px 0 0;
 }
 
-.subscriber_list_add .form-inline {
-    margin-bottom: 0;
-}
-
```

Comment:
Use already defined classname `.subscriber_list_add` to add CSS for `form-inline`.

For reference ->